sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -421,7 +424,11 @@ class AnySupport(
    */
   def decodeMessage(any: ScalaPbAny): Any = {
     val typeUrl = any.typeUrl
-    if (typeUrl.equals(BytesPrimitive.fullName)) {
+    val javaPbAny = ScalaPbAny.toJavaProto(any)
+
+    if (serializer.canDeserialize(javaPbAny)) {
+      serializer.deserialize(javaPbAny)
```

Comment:
This `canDeserialize` call is now applied once for every incoming request, event even though the feature is only for entity state (unless I missed some logic somewhere).

I think it would be good if it was more explicitly toggled at call site, something like `decodeMessage(any, supportJson = true)`.